role/token.go

```diff
@@ -28,14 +28,16 @@ import (
 // Token represents role token data.
 type Token struct {
 	// Version    string   // required
-	Domain string   // required
-	Roles  []string // required
-	// Principal  string   // required
+	Domain    string   // required
+	Roles     []string // required
+	Principal string   // required
 	// Host       string
 	// Salt       string    // required
 	// TimeStamp  time.Time // required
-	ExpiryTime time.Time // required
-	KeyID      string    // required
+	IntTimeStamp  int64     // required
+	ExpiryTime    time.Time // required
+	IntExpiryTime int64     // requir_ed
```

Comment:
```suggestion
```
better to just have `ExpiryTime` and convert in the getter with `ExpiryTime.Unix()`?
same for `TimeStamp`

Reply:
Fixed
https://github.com/yahoojapan/athenz-authorizer/pull/71/commits/8f2e7c54a9c8571f6eadc3e9cac6ab3d97f302ec